AWS/legos/aws_create_bucket/aws_create_bucket.py

```diff
@@ -14,7 +14,8 @@ class InputSchema(BaseModel):
         description='Name of the bucket to be created.')
     acl: str = Field(
         title='ACL',
-        description='The Canned ACL to apply to the bucket. Possible values: private, public-read, public-read-write, authenticated-read.')
+        description=('The Canned ACL to apply to the bucket. Possible values: '

```

Comment:
I am interested in the pylint error reported here, want to see if there is a better way of fixing this.